e2e/_suites/fleet/stand-alone.go

```diff
@@ -47,10 +47,18 @@ func (sats *StandAloneTestSuite) afterScenario() {
 	}
 
 	if _, err := os.Stat(sats.AgentConfigFilePath); err == nil {
-		os.Remove(sats.AgentConfigFilePath)
-		log.WithFields(log.Fields{
-			"path": sats.AgentConfigFilePath,
-		}).Debug("Elastic Agent configuration file removed.")
+		beatsLocalPath := shell.GetEnv("BEATS_LOCAL_PATH", "")
+		if beatsLocalPath == "" {
+			os.Remove(sats.AgentConfigFilePath)
```

Comment:
Do not remove files from the local repository

Reply:
I made a comment in #955, perhaps we should think about keeping `~/.op/compose` fully controlled by us and allow developers to utilize like a `~/.op/compose.d` for anything custom?